lib/usd/translators/meshWriter_BlendShapes.cpp

```diff
@@ -315,14 +311,12 @@ MStatus mayaGetBlendShapeInfosForMesh(
         // NOTE: (yliangsiew) Because this can end up grabbing the wrong plug (i.e.
         // blendShape.weight[1]), we double-check here and advance the iterator if necessary.
         if (!outputGeomElemPlug.isElement()) {
-            itDg.next();
```

Comment:
This is inside a for loop which already calls .next(). 
My understand is that there might be a hidden bug here where we can accidentally skip some elements.